nrefocus/roi_handling.py

```diff
@@ -0,0 +1,63 @@
+import numbers
+
+
+class ROIValueError(ValueError):
+    pass
+
+
+def parse_roi(roi):
+    """Handle the roi information.
+
+    Parameters
+    ----------
+    roi: list or tuple or ndarray or list of slices
+        Region of interest for which the metric will be minimized. The below
+        axes below are numpy axes. Options are:
+        list or tuple or numpy indexing array (old behaviour):
+            [axis_1_start, axis_0_start, axis_1_end, axis_0_end]
```

Comment:
See above. Is that really the same as `(slice(roi[0], roi[2]), slice(roi[1], roi[3]))`? I thought when you pass a tuple of slices, the first slice will be for axis 0 and not for axis 1...

Reply:
Yes you are right, see my reply above :)